core/chains/evm/logpoller/orm.go

```diff
@@ -839,5 +902,13 @@ func nestedBlockNumberQuery(confs Confirmations) string {
 			FROM evm.log_poller_blocks 	
 			WHERE evm_chain_id = :evm_chain_id 
 			ORDER BY block_number DESC LIMIT 1) `
+}
 
+func UseTopicIndex(index int) (int, error) {
```

Comment:
You already use `withTopicIndex` instead of this. You might want to delete it.

Reply:
Yup, don't need this anymore.